app/src/test/java/com/example/marcinwisniewski/intivetrainingapp/RxImmediateSchedulerRule.kt

```diff
@@ -0,0 +1,30 @@
+package com.example.marcinwisniewski.intivetrainingapp
+
+import io.reactivex.android.plugins.RxAndroidPlugins
+import io.reactivex.plugins.RxJavaPlugins
+import io.reactivex.schedulers.Schedulers
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+class RxImmediateSchedulerRule : TestRule {
```

Comment:
Do you have any links about this solution ? Was it practiced before ?

Reply:
I found similar solution on few programmers blogs and to be honest I haven't got better idea to handle that schedulers problem. I tried to read about it and correct it a little bit and this is effect of my work :)